chapters/introduction.tex

```diff
@@ -72,8 +72,33 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+The simulation starts with solving the initialization problem at the starting time giving values for all variables that are consistent with the result of the flattening.
```

Comment:
Some rewording and a comma:
```suggestion
Simulation starts with solving the initialization problem at the starting time, resulting in values for all variables that are consistent with the result of the flattening.
```

To me, "the result of the flattening" sounds a bit strange.  Wouldn't it be more important to stress the DAE aspect of the initialization problem than saying it comes out of flattening?

Reply:
I tried to keep the new terms to a minimum; so I prefer to keep 'flattening' here.